app/views/organizations/_disabled_organization_select.html.erb

```diff
@@ -0,0 +1,6 @@
+<% aria_label = org[:role] != 'admin' ? 'You are not an admin of this org' : 'This org has already been added to Classroom' %>
```

Comment:
> You are not an admin of this org

Make this more clear: "You cannot add this organization to classroom because you are not an owner"

Now the user will know to go get admin access from someone.

> This org has already been added to Classroom

What can the user do in this situation? Maybe better to say "Another owner has already added this organization to Classroom for GitHub"